scania-digital-design-system/sdds
# fix(sdds-link): refactored sdds-link

**Describe pull-request**  
Removed all "link-logic" from the sdds-link and instead created a default slot where people can slot in any link they want.
This will impact Banner and Toast.

**Solving issue**  
Fixes: #

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Link -> Web Component
3. Check the API and that the styling is correct.

**Suggested test steps**
